src/flb_log.c

```diff
@@ -717,6 +820,25 @@ void flb_log_print(int type, const char *file, int line, const char *fmt, ...)
 
     w = flb_worker_get();
     if (w) {
+        config = w->config;
+        if (config != NULL && config->log != NULL) {
+            msg_type_str = flb_log_message_type_str(type);
+            if (msg_type_str == NULL) {
+                msg_type_str = "unknown";
+            }
+
+            ts = cfl_time_now();
+            ret = cmt_counter_inc(config->log->metrics->logs_total_counter,
+                                  ts,
+                                  1, (char *[]) {msg_type_str});
+            if (ret == -1) {
+                // Not using flb_log_debug to avoid recursing into this same function.
```

Comment:
Please replace this with a C style comment `/**/`

Reply:
Done.